### PR TITLE
Add OTEL gateway + bundle KSM + chart improvements

### PR DESCRIPTION
**Pull request by dbudziwojskiNR**

### CODE REVIEW

File: otel-agent-chart/templates/deployment-configmap.yaml [hunk 1]
```diff
@@ -35,13 +32,37 @@ data:
...
+        sampling_thereafter: 200
     service:
       telemetry:
         logs:
           level: "debug"
       pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [otlp/newrelic, debug]
```
Comment (csongnr) [inline otel-agent-chart/templates/deployment-configmap.yaml]: no traces

Comment (dbudziwojskiNR) [reply otel-agent-chart/templates/deployment-configmap.yaml]: Should I just delete the traces pipeline? This is part of the gateway config so maybe customers might forward traces to the gateway for NR to ingest? If not, we can delete it.

Comment (csongnr) [reply otel-agent-chart/templates/deployment-configmap.yaml]: yes, just delete

Comment (csongnr) [reply otel-agent-chart/templates/deployment-configmap.yaml]: actually, you can comment it out and leave a note, (#uncomment if exporting traces to NR)

**Review by csongnr — APPROVED** [review]

**Pull request merged**